CHANGELOG.md

```diff
@@ -49,15 +50,26 @@ and removing some features that no longer seem necessary.
   browser normalization, is no longer included. We now recommend
   using [CSS Remedy](https://github.com/jensimmons/cssremedy).
```

Comment:
Do we want to address the spacing issues that we are seeing on projects that are using this init-free version of Accoutrement?

Reply:
I think it could be worth it to add a note mentioning the specific layout issues we've noticed, and/or a general note saying "this might break layouts" . . . I'll try to gather what we've noticed so far and add that in.